docs/ipv6-support.md

```diff
@@ -0,0 +1,42 @@
+## Overview
```

Comment:
Is this a new document that we are adding in 1.9.2?

Reply:
yes, this is the support matrix for IPv6

Reply:
@aaha97 : Please mention end to end support is there only for L7.

Reply:
added statement on end to end support